src/libAtomVM/externalterm.c

```diff
@@ -181,7 +181,7 @@ term externalterm_to_binary(Context *ctx, term t)
     //
     // Ensure enough free space in heap for binary
     //
-    int size_in_terms = term_binary_data_size_in_terms(len);
+    int size_in_terms = term_binary_heap_size(len);
     if (UNLIKELY(memory_ensure_free(ctx, size_in_terms + 1) != MEMORY_GC_OK)) {
```

Comment:
I am not exactly sure what this +1 is doing here.  (I wrote it 4 years ago :derp:)

Reply:
Removed the `+ 1`